config/routes.rb

```diff
@@ -2,6 +2,13 @@
   Rails.application.routes.draw { mount RailsEventStore::Browser => "/res" if Rails.env.development? }
   mount DatastoreApi::HealthEngine::Engine => '/datastore'
 
+  # TODO: should this be accessible in prod? By which users?
```

Comment:
I think access should be internal, just wondering what it would display that other prod users would need? Also, is there an example on what it would look like/display?

Reply:
It would show information about how our jobs are doing, which ones are failing, etc. Lots of useful information in a nice UI. It would also allow you to easily kill and re-run jobs, so it would only be useful to and should be restricted to devs, realistically. A lot of that information would surface in the logs anyway and we can use Prometheus metrics to monitor it as well, so it's not essential but more of a convenience. I've seen other teams use basic username and password authentication.

![image](https://github.com/user-attachments/assets/62ad97a4-4c22-4baa-bd57-54936ddcbd57)

Reply:
Nice! probably useful for devs only to begin with and then we can give access to others if we feel it's necessary later